Update descriptions and parameters SetHeliBladesFullSpeed & SetHeliBladesSpeed (#1236)
citizenfx/natives: VEHICLE/SetHeliBladesFullSpeed.md

void SET_HELI_BLADES_FULL_SPEED(Vehicle vehicle);
```

> Sets the speed of the helicopter blades to full speed.

Reviewer: ```suggestion
Despite its name, this works on Helicopters and Planes.

Sets the speed of the helicopter blades to full speed.

```

> Equivalent of `SET_HELI_BLADES_SPEED(vehicleHandle, 1.0f);`.

Reviewer: ```suggestion
This is equivalent to calling `SetHeliBladesSpeed(vehicleHandle, 1.0);`
```

>   
> Works with planes too.

Reviewer: ```suggestion
```
